### PR TITLE
added template folder with example .github/workflows/action.yml

### PR DESCRIPTION
**Pull request by panzacoder**

### CODE REVIEW

**Review by spreston4 — APPROVED** [review]
LGTM

File: template/.github/workflows/shipwright-test.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  build:
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - name: Shipwright Tokens
+        uses: headwayio/shipwright-tokens@main
+        with:
+          styleSystem: mui
+          outputFolder: styles
```
Comment (cplumley) [inline template/.github/workflows/shipwright-test.yml:23]: Do you think there should be a comment here with the different options? Maybe something like:
```suggestion
        # Pass in options to the Github Action: styleSystem, outputFolder, etc.
        with:
          # Options: mui, scss, tailwind
          styleSystem: mui
          # This is relative to the root of the repo
          outputFolder: styles
```

File: template/.github/workflows/shipwright-test.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Shipwright Tokens Example
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  pull_request:
+    branches: ["main"]
```
Comment (cplumley) [inline template/.github/workflows/shipwright-test.yml:9]: I haven't looked into the methods available in this file. If you use `pull_request` does that mean the action runs on both push and pull? The comment makes it seem like that would be the case

File: template/.github/workflows/shipwright-test.yml (hunk 1 above)
Comment (panzacoder) [reply template/.github/workflows/shipwright-test.yml:23]: That's great! I'll commit these changes. 

I think we should also have more detailed documentation about what each option does and what needs to be setup in the main projects' README

File: template/.github/workflows/shipwright-test.yml (hunk 2 above)
Comment (panzacoder) [reply template/.github/workflows/shipwright-test.yml:9]: Oh the comment is leftover, I removed the "push" key

it used to look like 
```yaml
push:
  branches: ["main"]
pull_request:
  branches: ["main"]

**Pull request merged**